cargo-dist/src/init.rs

```diff
@@ -475,6 +479,8 @@ fn get_new_dist_metadata(
                 meta.tap = None;
             } else {
                 meta.tap = Some(tap.to_owned());
+                meta.publish_jobs = Some(vec![PublishStyle::Homebrew]);
```

Comment:
We should write this more compositionally. Start with the existing inputs read from the config, then push this value if it's not already in the array (will make things a lot better when we start adding more).

Reply:
in retrospect this one is ignorable if we just want to get it out the door

Reply:
Done.